helm_deploy/hmpps-prisoner-to-nomis-update/templates/report-locations-cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-report-locations
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "30 2 * * 3"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 600
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      # Tidy up all jobs after 4 days
+      ttlSecondsAfterFinished: 345600
+      template:
+        spec:
+          containers:
+            - name: report-locations
+              image: ghcr.io/ministryofjustice/hmpps-devops-tools
+              args:
+                - /bin/sh
+                - -c
+                - curl --retry 2 -XPUT http://hmpps-prisoner-to-nomis-update/locations/reports/reconciliation
+          restartPolicy: Never
```

Comment:
I've added a `securityContext` section to all the cronjobs since this PR was raised

Reply:
ok sorted 👍